packages/pwa-kit-dev/src/utils/script-utils.test.js

```diff
@@ -0,0 +1,300 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {mkdtemp, rmdir, writeFile} from 'fs/promises'
+
+const pkg = require('../../package.json')
+import * as upload2 from './upload2'
```

Comment:
```suggestion
import * as utils from './script-utils'
```
I think you missed a spot 😉